src/Extensions/Promise.test.ts

```diff
@@ -40,4 +41,18 @@ describe('Promise', async (should: any) => {
       expected: ERROR,
     })
   }
+
+  {
+    const actual = await Promise.reject(10)
+      .rethrow(error => error === 10 && TRANSLATION)
+      .then(success)
+      .catch(error => error)
+
+    assert({
+      given: 'Promise.reject(10).rethrow(10)',
+      should: 'reject',
```

Comment:
Maybe `should: 'reject with translated error`?

Reply:
yep